src/commonMain/kotlin/dev/romainguy/kotlin/math/Matrix.kt

```diff
@@ -212,6 +220,8 @@ data class Mat4(
     constructor(m: Mat4) : this(m.x.copy(), m.y.copy(), m.z.copy(), m.w.copy())
 
     companion object {
+        val DEFAULT_ROTATIONS_ORDER = RotationsOrder.ZYX
```

Comment:
This is a constant pointing to another constant, this seems completely unnecessary.

Reply:
So we cannot define the default rotations order at the companion level?

Reply:
Why would this be needed?

Reply:
For the user to define it statically at the Matrix level instead of on each call?

Reply:
But it's true that it's not Matrix but function related.
So I rollback.

Reply:
You expect users to change the source code?

Reply:
LOL. No, but I should definitely stop coding with Swift because It sometimes makes me doing mistakes between `let`, `val `and `var`